great_expectations/datasource/fluent/sources.pyi

```diff
@@ -320,20 +320,23 @@ class _SourceFactories:
         *,
         spark_config: SparkConfig | None = None,
         force_reuse_spark_context: bool = True,
+        persist: bool = True,
     ) -> SparkDatasource: ...
     def update_spark(
         self,
         name: str,
         *,
         spark_config: SparkConfig | None = None,
         force_reuse_spark_context: bool = True,
+        persist: bool = True,
     ) -> SparkDatasource: ...
     def add_or_update_spark(
         self,
         name: str,
         *,
         spark_config: SparkConfig | None = None,
         force_reuse_spark_context: bool = True,
+        persist: bool = True,
```

Comment:
Can you also add this parameter to the `*_spark_filesystem`, `*_spark_dbfs`, `*_spark_s3`, `*_spark_gcs`, and `*_spark_abs` methods in this file?

Reply:
Nice callout, applied.
Anyway, I have also tested without adding persist as a clear parameter and it also works, as we have a * there.